src/main/java/com/cvsgo/controller/ImageController.java

```diff
@@ -0,0 +1,28 @@
+package com.cvsgo.controller;
+
+import com.cvsgo.dto.SuccessResponse;
+import com.cvsgo.service.FileUploadService;
+import java.io.IOException;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/images")
+public class ImageController {
+
+    private final FileUploadService fileUploadService;
+
+    @PostMapping("/{folder}")
```

Comment:
상태코드가 `200 OK`가 아닌 `201 CREARED` 가 적절할 것 같아요!
```suggestion
    @PostMapping("/{folder}")
    @ResponseStatus(HttpStatus.CREATED)
```